line_profiler/_line_profiler.pyx

```diff
@@ -156,72 +203,75 @@ cdef class LineProfiler:
 
     def enable(self):
         PyEval_SetTrace(python_trace_callback, self)
-
-    def disable(self):
-        self.last_time = {}
+        
+    @property
+    def code_map(self):
+        return <dict>self.c_code_map
+        
+    @property
+    def last_time(self):
+        return <dict>self.c_last_time
+
+    cpdef disable(self):
+        self.c_last_time.clear()
         unset_trace()
 
-    def get_stats(self):
+    cpdef get_stats(self):
```

Comment:
Is `get_stats` ever called from C? Is there any benefit of making a `cpdef` instead of a `def`?

Reply:
Whoopsie, I think I forgot to undo that. I was testing performance impacts a while back and it must have slipped into my commits.